video-library/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3.8'
+services:
+  db:
+    image: postgres:16.1-alpine
+    restart: unless-stopped
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgresSecret
+      - POSTGRES_DB=video-database
+      - PGDATA:/var/lib/postgresql/data/pgdata
+    ports:
+      - '5432:5432'
+    volumes: 
+      - .my_postgres_data:/var/lib/postgresql/data 
+    networks:
+      - my-bridge-network
```

Comment:
tohle nepotřebuješ, pokud je jen jeden network, tak to bude default pro všechny když jinak neřekneš